src/index.js

```diff
@@ -0,0 +1,476 @@
+'use strict';
```

Comment:
The way github is displaying this isn't great (it doesn't realize the file move/rename and changes). Here is the separate commit with whitespace ignored (`?w=1` flag) https://github.com/github/fetch/pull/616/commits/a5ec27348af874916d860f611b6f166aaad9ae5a?w=1#diff-1fdf421c05c1140f6d71444ea2b27638